data/free worlds start.txt

```diff
@@ -1615,6 +1821,116 @@ mission "FW Pirates 4.1"
+			`	"We're the ones asking the questions here!" Ariana snaps. `
+			`	One of the other Senators, an older man, asks, "Why did you choose to take part in the attack?"`
+			choice
+				`	"I wanted to end the pirate threat."`
+				`	"I trusted that Tomek would know the best strategy."`
+				`	"I guess I got too caught up in trying to be a hero."`
+					goto hero
+			`	"And how did that work out?" says the old man. "How long after the battle did it take you to realize that we have no way of holding on to the planet you conquered?"`
+			`	"Only Tomek said anything about conquering." you protest. "I saw the futility of annexing Greenrock as soon as I saw the planet up close. I never made any claim to the planet. I simply demanded they give up their slaves and get out of my way as I destroyed their piracy infrastructure.`
+			`	"Yet still, you attacked that pirate base. You disobeyed Senate orders, you disobeyed the Council's orders, and now you freely admit that you even disobeyed the rogue Council member who ordered the illegal strike. You took an oath of loyalty, and your actions show no loyalty to any proper authority."`
+			`	You bristle in defiance. "I became a starship captain because I yearned for freedom. I joined The Free Worlds because I believe in freedom. I have risked my life in defense of that freedom many times, and I intend to go on risking my life for The Free Worlds. I attacked Greenrock to free us from its constant piracy. I freed its slaves. I left Greenrock free behind me. The cause of freedom is the highest authority, and if you cannot respect what I've done, then the Senate of The Free Worlds is unworthy of its name."
+			`	A confused roar breaks out. A number of senators cry out their disapproval, but an equal number of senators thump their armrests or clap in approval. The older senator purses his lips, rubs his chin, and furrows his brow. Ariana scowls.
+				goto sentence
+			label hero
+			`	"Indeed," says the senator. "And I bet it wasn't long after the battle that you realized saving the galaxy isn't that easy."`
+			`	You shake your head. "It was never about being a hero to the galaxy. It was about being a hero to those slaves. Many of them have been waiting for generations to escape to a chance at a better life."`
+			`	The old senator blinks. "And you think you can provide them a better life?"`
+			`	You shrug. "I'm hoping they can be their own heroes from now on, but I'd be happy to help them find their way. They know where to find me."`
+			label sentence
+			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`
+			`	You are led back out to the lobby, where you wait for hours. Finally, a messenger comes and leads you to the Senator Sara's office. `
```

Comment:
Ok, I've overhauled the fluff text in the new Senate Hearing mission. I made the player's voice much less participatory in both the accept and arrival parts. This should make these missions more compliant with Endless Sky standards, allowing the player to have their own reactions without railroading them. I've also toned down the jerky side of JJ on Longjump, and rather than have the player defend him or herself, I have JJ come to the senate hearing and speak in the player's defense (appropriate since the player saved his sister, and it helps mend the player-JJ relationship). I've also given Senator Sara a last name, and I added some fluff to explain the presence of an Alpha carrier.